cockatrice/src/player.cpp

```diff
@@ -1601,15 +1602,15 @@ void Player::actUntapAll()
 
 void Player::actRollDie()
 {
-    bool ok;
-    int sides = QInputDialog::getInt(game, tr("Roll die"), tr("Number of sides:"), defaultNumberDieRoll, minDieRoll,
```

Comment:
will the new dialog function the same as the old getInt dialog? ie focus the number of sides and accept on hitting enter?

Reply:
Added